author.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Author
+  attr_reader :items
+
+  def initialize(first_name, last_name)
+    @id = rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @itmes << item
```

Comment:
- [ ] There is a typo here `@itmes` should be `@items`

Reply:
ooooh for sure I will fix it immediately thanks for your professional review